SCClassLibrary/Common/GUI/tools/Stethoscope.sc

```diff
@@ -437,6 +437,16 @@ Stethoscope {
 		);
 		^true;
 	}
+	
+	bounds_ { arg rect;
+		if( window.notNil ) {
+			if (rect.width < 264) {
+				"The width value you set will be changed to 264, the minimum width.".postln;
```

Comment:
Not a request for change, but I'm wondering if we have guidance on when we do/don't post warnings or informative notes (and considering this clipped minimum is documented in the Help file). @telephon

Reply:
Hm, there is no general guidance. Things to keep in mind: 
- if things may happen repeatedly, it is not good to repeatedly post warnings
- sometimes it is ok just not to warn and rely on an educated user